BitwardenShared/Core/Platform/Services/TestHelpers/MockStateService.swift

```diff
@@ -42,6 +42,7 @@ class MockStateService: StateService { // swiftlint:disable:this type_body_lengt
     var introCarouselShown = false
     var isAuthenticated = [String: Bool]()
     var isAuthenticatedError: Error?
+    var isEligibleForReviewPrompt = false
```

Comment:
⛏️ I think this, `resetUserActionCountsCalled` and `isEligibleForReviewPrompt()` aren't needed anymore?